.github/workflows/test-native.yml

```diff
@@ -92,5 +92,6 @@ jobs:
           ./mvnw install ${MAVEN_FAST_INSTALL} -pl '!presto-docs,!presto-server,!presto-server-rpm'
 
       - name: Run e2e tests
-        rm -rf /tmp/hive_data/tpch/
-        run: ./mvnw test ${MAVEN_TEST} -pl 'presto-native-execution' -DPRESTO_SERVER=${GITHUB_WORKSPACE}/presto-native-execution/_build/debug/presto_cpp/main/presto_server -DDATA_DIR=/tmp/ -Duser.timezone=America/Bahia_Banderas
+        run: |
+          rm -rf /tmp/hive_data/tpch/
+          ./mvnw test ${MAVEN_TEST} -pl 'presto-native-execution' -DPRESTO_SERVER=${GITHUB_WORKSPACE}/presto-native-execution/_build/debug/presto_cpp/main/presto_server -DDATA_DIR=/tmp/ -Duser.timezone=America/Bahia_Banderas
```

Comment:
@majetideepak 
What was the actual problem with the previous code?
What failed?
And how did it pass the tests, I wonder?

Reply:
@spershin The previous code added `rm -rf /tmp/hive_data/tpch/` in the middle of the workflow file. This is not valid.
And this caused 
```
[Invalid workflow file: .github/workflows/test-native.yml#L95](https://github.com/prestodb/presto/actions/runs/3098126436/workflow)
You have an error in your yaml syntax on line 95
```
as reported here https://github.com/prestodb/presto/actions/runs/3098126436

The fix in this PR is to make this command part of the `run` workflow command.
This is now valid and the workflow started to run successfully.